copy/single.go

```diff
@@ -31,6 +31,8 @@ import (
 // imageCopier tracks state specific to a single image (possibly an item of a manifest list)
 type imageCopier struct {
 	c                          *copier
+	compressionFormat          *compressiontypes.Algorithm // Compression algorithm to use, if the user explicitly requested one, or nil.
+	compressionLevel           *int
```

Comment:
(Absolutely non-blocking: Maybe move it near the encryption field, so that the ”important” `manifestUpdates` and `src` come first. This works just fine.)

Reply:
Moved below.